### PR TITLE
add new post

### PR DESCRIPTION
**Pull request by caironoleto**

### CODE REVIEW

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+E eu respondi:
+
+<blockquote class="twitter-tweet"><p lang="pt" dir="ltr">No final do dia, as pessoas são tudo, tudo vai girar em torno delas. Ouça mais, fale menos, converse com todos. Deixa tecnologia para o time. Melhore/crie um sistema de contratações, seja inclusivo nesse ponto, afinal vc tem esse poder especificamente.</p>&mdash; Cairo Noleto (@caironoleto) <a href="https://twitter.com/caironoleto/status/1208894250647261185?ref_src=twsrc%5Etfw">December 22, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## No final do dia, as pessoas são tudo
+
+E tudo vai girar em torno delas. A partir desse momento, você vai ter que lidar com Pessoas Engenheiras de Software e com isso vem todos os problemas de comunicação que isso pode gerar:
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
Tudo vai girar em torno delas. A partir desse momento, você vai ter que lidar com Pessoas Engenheiras de Software e com isso vem todos os desafios de comunicação que podem acontecer:
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+- uma frase mal colocada e mal interpretada;
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
- Frases mal colocadas e mal interpretadas;
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+- uma frustação salarial não resolvida;
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
- frustrações salarias não resolvidas;
```

Comment (LucasCFerraz) [reply content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]: Or remove it

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+- uma expectativa não atendida;
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
- expectativas não atendidas;
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+- uma ansiedade para uma nova oportunidade;
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
- ansiedades por novas oportunidades não endereçadas;
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+- ...
+
+E essa lista é imensa. A partir do momento que você é responsável por um ou mais times de engenharia, é esse tipo de problema que você vai ter que resolver. Esse é seu novo trabalho. E para isso, poucos lugares/literatura vão te ajudar nesse ponto, mas, existe uma em especial que adoro recomendar: [The Manager's Path: A Guide for Tech Leaders Navigating Growth and Change](https://www.amazon.com/Managers-Path-Leaders-Navigating-Growth/dp/1491973897).
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
Essa lista é imensa. A partir do momento que você é responsável por um ou mais times de engenharia, é esse tipo de problema que você vai resolver. Esse é seu novo trabalho. E para isso, você precisará ler, ouvir e aprender das mais diversas formas. Existem algumas opções de literatura, mas nenhuma que tenha me ajudado tanto quanto o livro de Camille Fournier: [The Manager's Path: A Guide for Tech Leaders Navigating Growth and Change](https://www.amazon.com/Managers-Path-Leaders-Navigating-Growth/dp/1491973897).
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+Seu foco de estudo precisa ser mais sobre pessoas do que sobre tecnologia. Você vai precisar traçar um caminho que, provavelmente, você nunca tinha se imaginado. Mas, assim como programação, você só precisa se dedicar e estudar muito, sem parar, assim você obtém sucesso!
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
Seu foco de estudo precisa ser mais sobre pessoas do que sobre tecnologia. Você vai precisar traçar um caminho que, provavelmente, você nunca tinha se imaginado. Mas, assim como programação, você só precisa se dedicar muito as mais diversas formas de aprendizado, sem parar, assim você terá sucesso!
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+## Ouça mais, fale menos, converse com todos
+
+Como as pessoas vão ser tudo para você daqui pra frente, você vai ter que começar a melhorar mais ainda as habilidades de comunicação. Por isso a dica para que você ouça mais, e aqui é tentar escutar tudo:
+
+- escute todas as pessoas do seu time;
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
- escute com todas as pessoas do seu time;
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+- escute todas as pessoas que interage com o seu time;
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
- escute todas as pessoas que interagem com o seu time;
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+- escute tudo que seus pares falam;
+- ligue o alerta em todas as reuniões que você for participar;
+- ouça pessoas de fora da companhia;
+- ouça um mentor;
+- ...
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
- sempre que possível, fale por último.
- ...
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+
+Uma vez que você escute tudo e todos, você tem embasamento para fazer melhores feedbacks e saber quais brigas você deve comprar, quando você deve intervir ou até mesmo quando você deve deixar uma pessoa errar para que ela aprenda por si só com o erro.
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
Uma vez que você escute tudo e todos, você tem embasamento para dar melhores feedbacks e saber quais brigas deve comprar, quando deve intervir ou até mesmo quando deixar uma pessoa errar para que ela aprenda por si só com o erro.
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+
+E uma vez que você está escutando mais, você naturalmente acaba falando menos. E o falar menos não é você não falar o que deve falar, mas saber quais palavras usar, como essas palavras vão ser interpretadas pelas outras pessoas e quando você deve falar o que precisa falar.
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
Uma vez que você escuta mais e com qualidade, naturalmente acaba falando menos. O falar menos não é você não falar o que deve ser dito, mas sim saber quais palavras usar, como essas palavras vão ser interpretadas e o momento.
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+
+Depois de escutar mais e ouvir menos, é hora de falar com todos! Você deve ter reuniões de one-on-one com todas as pessoas que reportam diretamente para você semanalmente e essa reunião é da pessoa, não é sua. Esse é o momento que você deve escutar tudo que a outra pessoa tem pra falar e também fazer feedbacks sobre ela.
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]: Some confusion in the start of this phrase

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 14]
```diff
@@ -0,0 +1,84 @@
...
+
+Não use one-on-ones para acompanhamento do trabalho da semana. Use daily meetings pra isso, elas são mais apropriadas. One-on-ones são reuniões que os participantes devem se preparar para participar, e tome nota de tudo. Não deixa de acompanhar tudo que acontece.
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
Não use one-on-ones para acompanhamento do trabalho da semana. Use daily meetings para isso, elas são mais apropriadas. One-on-ones são reuniões que os participantes devem estar preparados. Tome nota de tudo, não deixe de acompanhar ter um histórico do que acontece ao longo do tempo.
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 15]
```diff
@@ -0,0 +1,84 @@
...
+
+Estruture one-on-ones para falar sobre futuro, aspirações, evolução pessoal, onde ela quer chegar e mais tópicos nesse sentido. Use bem o tempo em que vocês vão conversar. Não fica mudando com frequência o horário dessa reunião, é uma tremenda falta de respeito.
+
+Além de one-on-ones regulares com seus diretos faça one-on-one com menos regularidade com as pessoas indiretas, e aí vão as pessoas engenheiras do time, POs, PMs e todo mundo que você precisa ouvir e dar feedbacks, uma vez que todos que estão próximos de você vão ser capazes de olhar para você com um segundo ponto de vista, assim aumentando mais ainda as possibilidades de feedback.
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
Além de one-on-ones regulares com seus diretos faça one-on-one com menos regularidade com as pessoas indiretas, e aí vão as pessoas engenheiras do time, POs, PMs e todo mundo que você precisa ouvir e dar feedbacks, uma vez que todos que estão próximos de você vão ser capazes de olhar para você com um segundo ponto de vista, aumentando as possibilidades de uma sugestão proveitosa.
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 16]
```diff
@@ -0,0 +1,84 @@
...
+
+## Deixe a tecnologia para o time
+
+Você não vai mais ter tempo para codar, essa é a verdade, logo, não seja o gargalo do time. Não assuma nenhuma responsabilidade de engenharia onde você vai ser o gargalo de outras pessoas, seja crítico e real: seu trabalho agora é ser responsável pelas pessoas e não mais ser o herói e fazer código. O código que você vai "codar" vai ser feito através do seu time e do que esse time é capaz e não mais de você diretamente.
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
Você não vai mais ter tempo para codar, essa é a verdade, logo, não seja o gargalo do time. Não assuma nenhuma responsabilidade de engenharia onde você vai ser um problema. Seja crítico e realista: seu trabalho agora é ser responsável pelas pessoas e não mais ser o herói e fazer código. O código que você vai "codar" vai ser feito através do seu time.
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 17]
```diff
@@ -0,0 +1,84 @@
...
+
+Dito isso, você deve saber quais tecnologias estão sendo usadas pelo time, até porquê você vai precisar contratar pessoas para esse time eventualmente, e guiar as decisões de tecnologia que seu time vai fazer. Você pode até travar a stack do time, mas com toda a certeza do mundo as pessoas vão querer usar coisas novas e legais em produção e você é a pessoa responsável para que isso aconteça.
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
Dito isso, você deve saber quais tecnologias estão sendo usadas pelo time, até porquê você vai precisar contratar pessoas eventualmente, e guiar as decisões de tecnologia que seu time vai tomar. Você pode até travar a stack do time, mas com toda a certeza do mundo as pessoas vão querer usar coisas novas e legais em produção e você é a pessoa responsável para que isso aconteça.
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 18]
```diff
@@ -0,0 +1,84 @@
...
+
+O seu papel quanto ao código no time é tirar todos os impedimentos que por ventura possa parecer, é negociar com as outras áreas prazos, entregas e liderar o time de engenharia para desenvolver novas features, apagar incêndios e controlar os danos que por ventura devem aparecer no dia a dia.
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
O seu papel quanto ao código no time é tirar todos os impedimentos que por ventura possam aparecer, negociando os prazos com as outras áreas, auxiliando na direção das entregas e liderarando o time de engenharia para desenvolver novas features, apagar incêndios e controlar os danos que por ventura devem aparecer no dia a dia.
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 19]
```diff
@@ -0,0 +1,84 @@
...
+
+Você vai guiar, vai pensar em como o time vai continuar aprendendo e aplicando coisas novas em produção, mas não seja autoritário e nem o dono da verdade. Use o que você tem de melhor, que é a sua equipe de engenharia.
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]: 🥇

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 20]
```diff
@@ -0,0 +1,84 @@
...
+
+## Melhore/crie um sistema de contratações, seja inclusivo nesse ponto
+
+Como seu trabalho é cuidar das pessoas, aumentar o time também é, e para isso você precisa fazer da forma certa. Gaste muito tempo pensando no melhor processo seletivo e em como você pode usar o processo para ter menos viés, ser mais inclusivo e assertivo com as contratações. O tempo gasto vai ser um tempo de qualidade, onde você estará pensando em como melhorar o processo.
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
Aumentar o time também é sua responsabilidade, e para isso você precisa contratar da forma certa. Gaste muito tempo pensando no melhor processo seletivo e em como você pode usar o processo para ter menos viés, ser mais inclusivo e assertivo com as contratações. O tempo gasto vai ser um tempo de qualidade, onde você estará pensando em como melhorar o processo.
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 21]
```diff
@@ -0,0 +1,84 @@
...
+
+Seja inclusivo, uma vez que um time diverso consegue olhar de outras formas para os problemas e assim aumenta as oportunidades de fazer tudo com mais qualidade. E pra se ter um time mais diverso, é preciso mais energia na hora da seleção.
```
Comment (LucasCFerraz) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
Seja inclusivo, uma vez que um time diverso consegue olhar de outras formas para os problemas e assim aumenta as oportunidades de fazer tudo com mais qualidade. Para se ter um time mais diverso, é preciso mais energia na hora da seleção.
```

Comment (LucasCFerraz) [reply content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]: Im reflecting about this last phrase.

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md (hunk 1 above)
Comment (betinaribeiro) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
E tudo vai girar em torno delas. A partir desse momento, você vai ter que lidar com Pessoas Engenheiras de Software e, com isso, vêm todos os problemas de comunicação que isso pode gerar:
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md (hunk 7 above)
Comment (betinaribeiro) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
Seu foco de estudo precisa ser mais sobre pessoas do que sobre tecnologia. Você vai precisar traçar um caminho no qual, provavelmente, você nunca tinha se imaginado. Mas, assim como programação, você só precisa se dedicar e estudar muito, sem parar, assim você obtém sucesso!
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 22]
```diff
@@ -0,0 +1,84 @@
...
+- uma ansiedade para uma nova oportunidade;
+- ...
+
+E essa lista é imensa. A partir do momento que você é responsável por um ou mais times de engenharia, é esse tipo de problema que você vai ter que resolver. Esse é seu novo trabalho. E para isso, poucos lugares/literatura vão te ajudar nesse ponto, mas, existe uma em especial que adoro recomendar: [The Manager's Path: A Guide for Tech Leaders Navigating Growth and Change](https://www.amazon.com/Managers-Path-Leaders-Navigating-Growth/dp/1491973897).
+
+Seu foco de estudo precisa ser mais sobre pessoas do que sobre tecnologia. Você vai precisar traçar um caminho que, provavelmente, você nunca tinha se imaginado. Mas, assim como programação, você só precisa se dedicar e estudar muito, sem parar, assim você obtém sucesso!
+
+## Ouça mais, fale menos, converse com todos
+
+Como as pessoas vão ser tudo para você daqui pra frente, você vai ter que começar a melhorar mais ainda as habilidades de comunicação. Por isso a dica para que você ouça mais, e aqui é tentar escutar tudo:
```
Comment (betinaribeiro) [inline content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
Como as pessoas vão ser tudo para você daqui pra frente, você vai ter que se preocupar muito com suas habilidades de comunicação. Por isso, a dica é para que você ouça mais. E aqui é tentar escutar tudo:
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md (hunk 8 above)
Comment (betinaribeiro) [reply content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]: I disagree

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md (hunk 14 above)
Comment (betinaribeiro) [reply content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
Não use one-on-ones para acompanhamento do trabalho da semana. Use daily meetings para isso, elas são mais apropriadas. One-on-ones são reuniões para as quais os participantes devem se preparar. Tome nota de tudo, não deixe de ter um histórico do que acontece ao longo do tempo.
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (betinaribeiro) [reply content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
O seu papel quanto ao código no time é tirar todos os impedimentos que por ventura possam aparecer, é negociar prazos e entregas com outras áreas e liderar o time de engenharia para desenvolver novas _features_, apagar incêndios e controlar os danos que eventualmente podem aparecer no dia a dia.
```

File: content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md [hunk 23]
```diff
@@ -0,0 +1,84 @@
...
+
+Essa frase é do livro [High Output Management](https://www.amazon.com.br/Output-Management-English-Andrew-Grove-ebook/dp/B015VACHOK/ref=asc_df_B015VACHOK/?tag=googleshopp00-20&linkCode=df0&hvadid=379725685153&hvpos=1o2&hvnetw=g&hvrand=16566403399515502424&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=1001629&hvtargid=pla-1004441367512&psc=1), do Andrew Grove, ex CEO e Chairman da Intel e ele resume muito bem em uma frase o que é o trabalho de desenvolvimento de software: um trabalho em conjunto, de equipe.
+
+A partir de agora seu trabalho vai ser o de **técnico**. Você precisa _inspirar_ o melhor do seu time para todos terem sucesso! Desenvolvimento de software é um trabalho coletivo, onde todos precisam estar no seu melhor, assim como qualquer [esporte coletivo](https://www.linkedin.com/pulse/desenvolvimento-de-software-%C3%A9-um-esporte-coletivo-jo%C3%A3o-daniel-duarte/).
```
[the remark this answers is not in the document]
Comment (betinaribeiro) [reply content/posts/que-conselho-voce-daria-para-alguem-que-acabou-de-se-tornar-gerente-de-engenharia.md]:
```suggestion
A partir de agora, seu trabalho vai ser o de **técnico**. Você precisa _inspirar_ o melhor do seu time para todas as pessoas terem sucesso! Desenvolvimento de software é um trabalho coletivo, em que cada integrante do time precisa estar no seu melhor, assim como em qualquer [esporte coletivo](https://www.linkedin.com/pulse/desenvolvimento-de-software-%C3%A9-um-esporte-coletivo-jo%C3%A3o-daniel-duarte/).
```

**Pull request merged**